### DISCUSSION TITLE
Where can I get the Hexagonal preset code for using it in Reactjs?

### DISCUSSION THREAD

**Discussion by Hardikbhanot** [matteobruni/tsparticles#2923] [Ideas]
Where can I get the Hexagonal preset code for using it in Reactjs?

**Comment by pushpenderkadian** [comment]
Yes I am facing the same issue

**Reply by ANWESHA2202** [reply]
The json isn't working properly

**Comment by ANWESHA2202** [comment]
The json isn't working properly

**Comment by matteobruni** [comment]
I've created a sample in CodeSandbox here: https://codesandbox.io/s/eloquent-estrela-h0n8f?file=/src/App.js

**Reply by Hardikbhanot** [reply]
Thanks for your help
